test/graphtilebuilder.cc

```diff
@@ -30,7 +30,8 @@ class test_graph_tile_builder : public GraphTileBuilder {
 void assert_tile_equalish(const GraphTile& a,
```

Comment:
maybe im misreading it but it looks like non of these changes were needed in this test

Reply:
ok